doc/source/ray-overview/installation.rst

```diff
@@ -223,16 +223,23 @@ on the AUR page of ``python-ray`` `here`_.
 Installing Ray with Anaconda
 ----------------------------
 
+.. note::
+
+  Ray is not officially distributed and supported on conda. While using a
+  conda environment, it is recommended to install Ray itself from PyPi using
+  `pip install ray` in the newly created environment.
```

Comment:
nit: Could we move the note to the end of the section?

Ray conda packages are maintained by the community, not the Ray team. While using a
  conda environment, it is recommended to install Ray from PyPi if possible.

Reply:
Done